File: clients/rust/src/systemd_unit_file_state.rs
// SystemdUnitFileState represents a SystemdUnitFileState model.
#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
pub enum SystemdUnitFileState {
    #[serde(rename="enabled")]
    Enabled,
    #[serde(rename="enabled-runtime")]
    EnabledMinusRuntime,
    #[serde(rename="linked")]
    Linked,
    #[serde(rename="linked-runtime")]
    LinkedMinusRuntime,
    #[serde(rename="masked")]
    Masked,
    #[serde(rename="masked-runtime")]
    MaskedMinusRuntime,
    #[serde(rename="static")]
    ReservedStatic,
    #[serde(rename="disabled")]
    Disabled,
    #[serde(rename="invalid")]
    Invalid,
}
impl Default for SystemdUnitFileState {
    fn default() -> SystemdUnitFileState {
        SystemdUnitFileState::Enabled
    }
}

File: clients/printnanny-os/typescript/src/SystemdUnitFileState.ts

enum SystemdUnitFileState {
  ENABLED = "enabled",
  ENABLED_MINUS_RUNTIME = "enabled-runtime",
  LINKED = "linked",
  LINKED_MINUS_RUNTIME = "linked-runtime",
  MASKED = "masked",
  MASKED_MINUS_RUNTIME = "masked-runtime",
  RESERVED_STATIC = "static",
  DISABLED = "disabled",
  INVALID = "invalid",
}
export { SystemdUnitFileState };